### PR TITLE
add beer api

### PR DESCRIPTION
**Pull request by erax83**
Beer Of The Week added to the welcome page with help of an API. Tell me if you think it is more correct to put the php code in a separate controller file.

### CODE REVIEW

**Comment by SimonNord** [comment]
Saw that it was a small bug that some of the results was missing the label property which caused an error message. 
Added a new parameter in the url to only get results with label images.

File: resources/views/welcome.blade.php [hunk 1]
```diff
@@ -12,6 +12,20 @@
...
         </ul>
+
+        <?php
+            $brewKey = "0cecc2de820dc46cde46804f597e1614";
+            $url = "http://api.brewerydb.com/v2/beers/?hasLabels=Y&key=$brewKey";
+            $databrew = file_get_contents($url);
+            $brew = json_decode($databrew);
+            $brewNum = rand(0, 45);
+            $brewPresent = ($brew->data[$brewNum]->name);
+            $brewPic = ($brew->data[$brewNum]->labels->icon);
```
Comment (andreasnyh) [inline resources/views/welcome.blade.php]: How do you feel about a bigger label picture?
```suggestion
            $brewPic = ($brew->data[$brewNum]->labels->medium);
```
It's possible to use 'labels->medium' or 'labels->large'

My vote is on medium =)

Comment (SimonNord) [reply resources/views/welcome.blade.php]: se

> How do you feel about a bigger label picture?
> 
> It's possible to use 'labels->medium' or 'labels->large'
> 
> My vote is on medium =)

Second that! Tried it out and it was much easier to see the label

Comment (erax83) [reply resources/views/welcome.blade.php]: Yes, much better.

**Review by andreasnyh — APPROVED** [review]

**Review by SimonNord — APPROVED** [review]

**Pull request merged**